com.unity.ml-agents/Runtime/Agent.cs

```diff
@@ -281,6 +295,24 @@ internal struct AgentParameters
         /// </summary>
         internal VectorSensor collectObservationsSensor;
 
+        /// <summary>
+        /// List of IActuators that this Agent will delegate actions to if any exist.
+        /// </summary>
+        ActuatorManager m_Actuators;
+
+        /// <summary>
+        /// DiscreteVectorActuator which is used by default if no other sensors exist on this Agent. This VectorSensor will
+        /// delegate its actions to <see cref="IActionReceiver.OnActionReceived"/> by default in order to keep backward compatibility
```

Comment:
Maybe I need to read more, but it sends to Agent.OnActionReceived, right?

Reply:
Yeah, Agent Implements that interface method.

Reply:
oh, I misunderstood this comment.  I'll fix it.